spec/game_validator_spec.rb

```diff
@@ -0,0 +1,122 @@
+require_relative './test_doubles/display_colour_double'
+require_relative './test_doubles/fake_class_double'
+require 'game_validator'
+require 'game_loader'
+require 'display'
+require 'yaml'
+
+def file_setup
+  filename = './spec/test_data/test.yml'
+  game_obj = 'game_object'
```

Comment:
Same as above, the names of variables shouldn't have their types in names eg. `game_string` or `game_obj`. If you want to indicate what the variable holds, you could say `game_data`.